### PR TITLE
Javadoc warning fixes.

### PR DESCRIPTION
**Pull request by ojacobson**
I've also reworded a couple of doc comments to be either clearer or more correct.

### CODE REVIEW

**Comment by ojacobson** [comment]
Surprisingly, I couldn't find a javadoc flag to turn warnings into errors, either in javadoc itself or [in Gradle](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.javadoc.Javadoc.html).

File: unacceptable-aliases/src/main/java/io/github/unacceptable/alias/UUIDGenerator.java [hunk 1]
```diff
@@ -17,13 +20,22 @@ public UUIDGenerator() {
...
     public UUID generate(final String alias) {
         return generator.generate(alias);
     }
 
     /**
-     * Generate a random UUID.
+     * @param alias
+     *         an alias for the generated value. The alias is unused in this generator, but can be used by an {@link
+     *         AliasStore} to associate the UUID with a name.
+     * @return a randomly-generated UUID.
```
Comment (lewisd32) [inline unacceptable-aliases/src/main/java/io/github/unacceptable/alias/UUIDGenerator.java]: That's a lot of duplicated text.  You don't want to use `@see` or something?

File: unacceptable-database/src/main/java/io/github/unacceptable/database/DatabaseContext.java [hunk 2]
```diff
@@ -201,7 +209,8 @@ public void destroy() throws SQLException {
 
     /**
      * Construct a URL for the test database. By default, this assumes that the admin URL's scheme-specific part is
-     * itself a URL, and that the path of that URL can be changed to any {@code /DATABASE-NAME}. This is appropriate for
+     * itself a URL, and that the path of that URL can be changed to any {@code /DATABASE-NAME}. This is appropriate
+     * for
      * PostgreSQL, MySQL, and several other databases.
```
Comment (lewisd32) [inline unacceptable-database/src/main/java/io/github/unacceptable/database/DatabaseContext.java]: Might want to wrap this line up.

**Comment by lewisd32** [comment]
lgtm, other than the one comment.

File: unacceptable-aliases/src/main/java/io/github/unacceptable/alias/UUIDGenerator.java (hunk 1 above)
Comment (ojacobson) [inline unacceptable-aliases/src/main/java/io/github/unacceptable/alias/UUIDGenerator.java]: I would love it if Javadoc had a tool similar to `{@inheritDoc}` for repeated documentation chunks, honestly.

Javadoc is terrible

**Pull request merged**